webassembly.html

```diff
@@ -226,9 +230,10 @@ <h1 id="title">[DRAFT] WebAssembly Working Group Charter</h1>
           <dl>
             <dt id="web-assembly" class="spec">WebAssembly</dt>
             <dd>
-              <p>This specification defines a portable, size- and
-              load-time-efficient format suitable for compilation to the
-              web.</p>
+              <p>This specification defines a size- and
+              load-time-efficient format and execution environment
+              that allow compilation to web with consistent behavior across a
```

Comment:
that allow compilation to web with consistent behavior across a


need “the web” here too